torch/_inductor/codecache.py

```diff
@@ -829,13 +833,12 @@ def map(cls, fn, seq):
             return list(map(fn, seq))
         return [t.result() for t in [cls.pool().submit(fn, x) for x in seq]]
 
-    def triton(self, source_code):
+    def triton(self, source_code, device="cuda"):
```

Comment:
In the future, Triton can support different devices. Hence, We need to add device information indicating Triton how to generate the code.